src/discord/commands/global/d.calc-benzo.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const calcBenzo = require('../../../global/utils/calc-benzo');
+const path = require('path');
+const fs = require('fs');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const _ = require('underscore');
+const logger = require('../../../global/utils/logger');
+const template = require('../../utils/embed-template');
+
+const PREFIX = path.parse(__filename).name;
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('calc-benzo')
+    .setDescription('Check combo information')
+    .addIntegerOption(option => option.setName('i_have')
+      .setDescription('mg')
+      .setRequired(true))
+    .addStringOption(option => option.setName('mg_of')
+      .setDescription('Pick the first benzo')
+      .setRequired(true)
+      .setAutocomplete(true))
+    .addStringOption(option => option.setName('and_i_want_the_dose_of')
+      .setDescription('Pick the second drug')
+      .setRequired(true)
+      .setAutocomplete(true)),
+
+  async execute(interaction, parameters) {
+    const dosage = interaction.options.getInteger('i_have') || parameters.at(0);
+    const drugA = interaction.options.getString('mg_of') || parameters.at(1);
+    const drugB = interaction.options.getString('and_i_want_the_dose_of') || parameters.at(2);
+
+    const data = await calcBenzo.calcBenzo(dosage, drugA, drugB);
+
+
+    const embed = template.embedTemplate()
+      .setColor('RANDOM')
+      .setTitle(`${dosage} mg of ${drugA} is ${data.result} mg of ${drugB}`)
+      .setDescription(`
+        This is a simple tool made to help you figure out how much of a given benzodiazepine dose converts into another benzodiazepine dose.\n\n\
+        **Please make sure to research the substances thoroughly before using them.**\n\n\
+        A good idea is to compare the effects of the two different benzodiazepines, as even though the dose is 'similiar' you might not get the effects you're used to.\n\n\
+        Important: Equivalent doses may be inaccurate for larger quantities of benzos with different effect profiles. Please compare the dosages below to see weighted dosage ranges.\n\n\
+        Note: It's a good idea to start with a lower dose than the calculator shows, since everybody can react differently to different substances.\n
+      `)
+      .addFields(
+        { name: `${drugA} Summary`, value: `${data.drugAResult.properties.summary}`, inline: true },
```

Comment:
Jul-19-2022 08:19:37 error:   [start] error.name: TypeError on line undefined 
Jul-19-2022 08:19:37 error:   [start] error.message: Cannot read properties of undefined (reading 'properties')
Jul-19-2022 08:19:37 error:   [start] error.stack: TypeError: Cannot read properties of undefined (reading 'properties')
    at Object.execute (C:\Projects\Discord\tripsit-discord-bot\src\discord\commands\global\d.calc-benzo.js:47:64)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
Jul-19-2022 08:19:37 error:   [start] error.code: undefined